web/api/badges/web_buffer_svg.c

```diff
@@ -862,19 +745,45 @@ void buffer_svg(BUFFER *wb, const char *label, calculated_number value, const ch
                 "<stop offset=\"1\" stop-opacity=\".1\"/>"
             "</linearGradient>"
             "<mask id=\"round\">"
-                "<rect width=\"%0.2f\" height=\"%0.2f\" rx=\"%0.2f\" fill=\"#fff\"/>"
+                "<rect class=\"bdge-ttl-width\" width=\"%0.2f\" height=\"%0.2f\" rx=\"%0.2f\" fill=\"#fff\"/>"
```

Comment:
Very cool that you added the classes, but what's the point without a css? Can I even use them? Maybe there's something I don't understand here...

Reply:
reason is javascript code below which actually updates widths according to real rendered text size in browser. So we guess size with verdana11_width but browser in the end updates the values to real measured ones. This will also then work if some user doesn't have veranda font (fallback font will be used) etc.etc.

Reply:
Cool! I missed the JS code.